get-docs-version-name/action.yml

```diff
@@ -0,0 +1,34 @@
+name: "get-docs-version-name"
+description: "Get the version name of the docs"
```

Comment:
Where does this description show up?
IMO, since we won't publish this to GitHub Marketplace, it's better to comment in README.md instead of this description.

Reply:
I mean, if we write these descriptions, we should make them more clear and easy to understand.
If not, I think it's better to remove these and write good READMEs instead to reduce the maintenance cost.

Here is an example of the good README.
https://github.com/tier4/github-actions/blob/21419e073219963876fab34be75814658c632814/ros-modified-packages/README.md

Reply:
???
This syntax is required.
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddescription

Reply:
Okay, I see. I thought it was optional because other workflows are probably so.
Then, we have to fix them so that people can understand.
Also anyway, we need READMEs separately, so could you add READMEs? Empty ones are acceptable for now.

Reply:
Or I feel making the description to be the same as the name is acceptable.

Reply:
> Then, we have to fix them so that people can understand.

I agree with you. The official GitHub documentation is confusing.

> Also anyway, we need READMEs separately, so could you add READMEs? Empty ones are acceptable for now.

It's good to understand for users. I will add README.

Reply:
Added readme
df9cc4a